src/index.ts

```diff
@@ -57,6 +59,7 @@ export function createNetworkFixture(
     async ({ page }, use) => {
       const worker = new NetworkFixture({
         page,
+        ignoreCommonAssetRequests: args?.ignoreCommonAssetRequests ?? false,
```

Comment:
I opted to make this an option and default it to `false` so as not to introduce a breaking change.

Reply:
I suggest drop this and make it `true` by default since you're introducing a breaking change anyway (`feat:`).